### PR TITLE
Adding console methods such as warn, info, dir

### PR DESCRIPTION
**Pull request by amandeepmittal**
- Console methods added: info, warn, dir
- updating minor version
- added contribution if PR accpeted
- Solves issue #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
   "name": "fancy-log",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (phated) [inline package.json]: please remove this bump.  We use `npm version ...` to version.

Comment (amandeepmittal) [reply package.json]: oh yes, sorry, of course.

**Comment by amandeepmittal** [comment]
@phated Done.

**Comment by phated** [comment]
I'm adding tests and doing some cleanup.  Publishing shortly.

**Comment by phated** [comment]
Little side note: `console.warn` outputs on `stderr` so I had to adjust the timestamp write to output there, like `error`.

**Comment by phated** [comment]
Published as 1.3.0

**Comment by phated** [comment]
Thanks for working on this @amandeepmittal and thanks for hanging in there until I had time to look at this.

**Comment by amandeepmittal** [comment]
Thank you for accepting my PR.

**Pull request merged**